### ISSUE TITLE
Add option in UI to change depth?

### ISSUE THREAD

**Issue by Ragarnoy** [abassibe/Gomoku#82]
Just realized maybe we don't want to recompile every time the correcteur wants to change the depth, opinion ?

**Comment by Ragarnoy** [comment]
Avis ? @demonoidv @tuturr @abassibe

**Comment by demonoidv** [comment]
Yep, I think we need an option for that, maybe we can even introduce the possibility to change the depth in the middle of a game?

**Comment by arlaine4** [comment]
I think the possibility of changing the depth for the game could be nice, but I don't see why we would want to change it in the middle of a game honestly
So I would say yes but beforehand and not during the game, at least that's my opinion

**Comment by arlaine4** [comment]
But I think most of the game the corrector won't try to change the depth, I mean we will probably tell him that the current depth is were the AI is the most efficient, so probably won't bother to try and change this parameter, and even if he does I mean that's what 10 to 20seconds compiling time, not a big deal
We should probably focus on more important stuff, if we want to implement new features before we push the project
